project/project/CakePlatformDependencies.scala

```diff
@@ -100,23 +100,23 @@ object CakePlatformDependencies {
   object SbtDependencies {
     val buildInfo: ModuleID = "com.eed3si9n" % "sbt-buildinfo" % "0.7.0"
     val digest: ModuleID = "com.typesafe.sbt" % "sbt-digest" % "1.1.3"
-    val dynver: ModuleID = "com.dwijnand" % "sbt-dynver" % "1.2.0"
-    val gatling: ModuleID = "io.gatling" % "gatling-sbt" % "2.2.1"
+    val dynver: ModuleID = "com.dwijnand" % "sbt-dynver" % "1.3.0"
+    val gatling: ModuleID = "io.gatling" % "gatling-sbt" % "2.2.2"
```

Comment:
I think this gatling dependency can go from here. Its the sort of thing that should be enforced at the template or project level.

Reply:
agree but used by the `PerformanceTestPlugin` so should be addressed with https://github.com/cakesolutions/sbt-cake/issues/64